pkgs/games/pegasus-frontend/default.nix

```diff
@@ -2,41 +2,37 @@
 , fetchFromGitHub
 , stdenv
 , cmake
-, qtbase
-, qtgraphicaleffects
-, qtmultimedia
-, qtsvg
-, qttools
-, qtx11extras
+, libsForQt5
```

Comment:
Why?

Reply:
I mistakenly tried this file outside using callPackage instead of libsForQt5.callPackage and thought "where is all of the qt related packages". Now I now why this doesn't work. Will revert back to using qtbase etc. from the arguments.

thanks